tests/components/ourgroceries/test_coordinator.py

```diff
@@ -0,0 +1,40 @@
+"""Define tests for the OurGroceries coordinator."""
```

Comment:
I think you can move these tests to the Todo tests for style. This isn't a blocker tho, just personal preference